### PR TITLE
[ENH, MRG] Add function references

### PR DESCRIPTION
**Pull request by alexrockhill**
Adds function references per @janfreyberg review comment.

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -50,28 +50,30 @@ Intracranial electrophysiology analysis requires precise location and anatomical
...
 
-To integrate these electrophysiology and imaging data, first, the CT must be aligned to the MR in order to determine the positions of the electrode contacts in the CT relative to brain structures in the MR. In MNE-Python, these are aligned using a mutual information histogram-matching approach using DIPY [@GaryfallidisEtAl2014] that works without manual pre-alignment.
+To integrate these electrophysiology and imaging data, first, the CT must be aligned to the MR in order to determine the positions of the electrode contacts in the CT relative to brain structures in the MR. In MNE-Python, these are aligned using a mutual information histogram-matching approach using DIPY [@GaryfallidisEtAl2014] that works without manual pre-alignment using the function ``mne.transforms.apply_volume_registration``.
```
Comment (larsoner) [inline paper.md:53]: why double backticks? single should be enough for markdown

Comment (alexrockhill) [reply paper.md:53]: Ah dang, that's how I had it originally but I thought it was wrong so I changed it, I'll revert

**Comment by larsoner** [comment]
You can just use `[ci skip]` for these commits, we don't want any of them to run (including CircleCI)

File: paper.md [hunk 2]
```diff
@@ -50,28 +50,30 @@ Intracranial electrophysiology analysis requires precise location and anatomical
...
 
 ![Figure 6. a) The anatomical labels for an sEEG electrode shaft are shown as the contacts progress from deep (starting with 1) to superficial regions. b) The anatomical surfaces that sEEG electrodes pass through are rendered along with the trajectory of the electrode shaft. c) A time-frequency decomposition of the ECoG data rendered on top of a 3D image of the brain showing the location of the grid implant.](figures/Figure_6.png)
 
+These intracranial location and analysis steps shown in the MNE-Python tutorials "Locating intracranial electrode contacts", "Working with sEEG data" and "Working with ECoG data".
```
Comment (drammock) [inline paper.md:75]:
```suggestion
These intracranial location and analysis steps are shown in the MNE-Python tutorials "Locating intracranial electrode contacts", "Working with sEEG data" and "Working with ECoG data".
```

**Comment by alexrockhill** [comment]
Thanks!

**Pull request merged**